app/services/sockethub-irc.js

```diff
@@ -92,7 +92,6 @@ export default Ember.Service.extend({
   handleJoinCompleted(space, message) {
     var channel = space.get('channels').findBy('sockethubChannelId', message.target['@id']);
     if (!isEmpty(channel)) {
```

Comment:
Shouldn't this be `isPresent` instead (and also be made the same code as above where it's just `is(channel)`)?